src/structures/Sticker.js

```diff
@@ -72,16 +105,62 @@ class Sticker extends Base {
     return new Date(this.createdTimestamp);
   }
 
+  /**
+   * Whether this sticker is partial
+   * @type {boolean}
+   * @readonly
+   */
+  get partial() {
+    return 'type' in this;
```

Comment:
Shouldn't this be 
```suggestion
    return !('type' in this);
```

Reply:
also not sure if it was intended but partial will always be false since the `type` property is always being added to the class.

Reply:
yeah i'll redo this